### PR TITLE
Add ESDT UTK token.

### PR DESCRIPTION
**Pull request by zdenal**
### 📝 Description

Added the Utrust (UTK) token to the esdt.js file since it has been added to the Elrond Network. Recreated PR from https://github.com/LedgerHQ/ledger-live/pull/535

### ❓ Context

- **Impacted projects**: `ledger-live`
- **Linked resource(s)**: ``

### ✅ Checklist

- [x] **Test coverage**
- [x] **Atomic delivery**
- [x] **No breaking changes**

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: libs/ledger-live-common/src/currencies/__snapshots__/sortByMarketcap.test.ts.snap [hunk 1]
```diff
@@ -6597,6 +6597,7 @@ Array [
   "algorand/asa/137594422",
   "elrond/esdt/555344432d633736663166",
   "elrond/esdt/524944452d376431386539",
+  "elrond/esdt/55544B2D326638306539",
```
Comment (zdenal) [inline libs/ledger-live-common/src/currencies/__snapshots__/sortByMarketcap.test.ts.snap:6600]: updated snapshot for test

**Comment by zdenal** [comment]
getting error on test `[Test] Desktop App / Ledger Live Desktop Tests (windows-latest) (pull_request)`. Is there anything what I can do w/ this?

```
/blake2@4.1.1/node_modules/blake2 install: D:\a\ledger-live\ledger-live\node_modules\.pnpm\blake2@4.1.1\node_modules\blake2\src\binding.cpp(1,10): fatal  error C1083: Cannot open include file: 'node.h': No such file or directory [D:\a\ledger-live\ledger-live\node_modules\.pnpm\blake2@4.1.1\node_modules\blake2\build\binding.vcxproj]
```

**Comment by zdenal** [comment]
@haammar-ledger @aelmilligy-ledger we reopened new PR here instead of this one https://github.com/LedgerHQ/ledger-live/pull/535.

Can you please advice what we can do next?

Thanks a lot ;)

**Comment by aelmilligy-ledger** [comment]
Hi @zdenal, 

Thank you for your contribution.
Could you: 
- Have a look on the tests that are not passing the CI, 
- Add changeset (with pnpm changeset),
- Fix the conflicting files.

Best regards.

**Pull request merged**